src/foam/demos/olympics/Medal.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.ENUM({
+  package: 'foam.demos.olympics',
+  name: 'MedalColor',
+  values: [
+    {
+      name: 'GOLD',
+      label: 'Gold'
+    },
+    {
+      name: 'SILVER',
+      label: 'Silver'
+    },
+    {
+      name: 'BRONZE',
+      label: 'Bronze'
+    }
+  ]
+});
+
+foam.CLASS({
+  package: 'foam.demos.olympics',
+  name: 'Medal',
+
+  properties: [
+    { name: 'id', hidden: true },
+    { class: 'Int', name: 'year', shortName: 'y' },
+    {
+      class: 'Enum',
+      of: 'foam.demos.olympics.MedalColor',
+      name: 'color',
+      value: 'Gold',
+      shortName: 'c',
+      aliases: [ 'colour', 'medal' ],
+      tableFormatter: function(medal, e) {
+        return e.E('span').cssClass(medal.color.label).add(medal.color.label);
+      },
+      comparePropertyValues: function(o1, o2) {
```

Comment:
This can indeed be skipped; `Enum`s sort by their ordinals by default.

I guess the medals didn't need to be an enum, but it's handy to get their labels and now sort order from the enum. Also trying to use enums here has been shaking down bugs in the query parser, filter controller and elsewhere, so it's a useful test in any case.